src/season/season.ts

```diff
@@ -0,0 +1,19 @@
+import { SeasonStatus } from './seasonStatus';
+
+export class Season {
+    id: String
+    name: String
+    status: SeasonStatus
+
+    constructor(id: String, name: String) {
```

Comment:
When the constructor is used just to assign properties we can take a shortcut and do it like:

```ts
export class Season {
  constructor(public id: String, public name: String, public status = SeasonStatus.Upcoming) {}
}
```

The received properties get directly assigned to the instance 🙂

Reply:
Awesome, the more you know!